diesel/src/query_source/mod.rs

```diff
@@ -13,13 +13,6 @@ use query_builder::*;
 pub use self::joins::JoinTo;
 pub use self::peano_numbers::*;
 
-#[cfg(feature = "with-deprecated")]
-#[deprecated(since = "1.1.0", note = "Use `deserialize::Queryable` instead")]
-pub use deserialize::Queryable;
```

Comment:
Removing this breaks the CI for diesel with the postgres feature enabled.

Reply:
Whoops this is the one I meant to leave in